daprdocs/content/en/operations/components/setup-pubsub/pubsub-namespaces.md

```diff
@@ -1,14 +1,23 @@
 ---
 type: docs
-title: "Pub/Sub and namespaces"
-linkTitle: "Kubernetes namespaces"
+title: "How-To: Configure Pub/Sub components with multiple namespaces"
+linkTitle: "How-To: Configure with multiple namespaces "
```

Comment:
This linkTitle is too long. Can we make it shorter? Perhaps "How-To: Multiple namespaces"

Reply:
We should drop the "How-To:" part as well. I suggest "Multiple namespaces"

Reply:
so changed to like this

title: "HowTo: Configure Pub/Sub components with multiple namespaces"
linkTitle: "Multiple namespaces"